### ISSUE TITLE
ModuleNotFoundError: No module named 'goodreads_api_client.resources' when importing goodreads_api_client

### ISSUE THREAD

**Issue by cosmicatech** [mdzhang/goodreads-api-client-python#9]
I'm new to this api but this error comes up on import. It's probably me not satisfying requirements or something but it looks like an error with the code.

```
Python 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 07:18:10) [MSC v.1900 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import goodreads_api_client
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\-\AppData\Local\Programs\Python\Python36-32\lib\site-packages\goodreads_api_client\__init__.py", line 10, in <module>
    from goodreads_api_client.client import Client
  File "C:\Users\-\AppData\Local\Programs\Python\Python36-32\lib\site-packages\goodreads_api_client\client.py", line 5, in <module>
    import goodreads_api_client.resources as resources
ModuleNotFoundError: No module named 'goodreads_api_client.resources'
```

**Comment by pietermarsman** [comment]
I get the same error

**Comment by pietermarsman** [comment]
I wrote a fix for this: #11
